releasenotes-builder/src/main/java/com/github/checkstyle/github/NotesBuilder.java

```diff
@@ -66,6 +66,18 @@
             + "^doc: release notes.*[\r\n]?$|"
             + "^(config:|minor:|infra:|dependency:)(.|\n)*[\r|\n]?$");
 
+    /** String format pattern for github issue. */
+    private static final String GITHUB_ISSUE_TEMPLATE =
+            " https://github.com/checkstyle/checkstyle/issues/%d";
```

Comment:
Why can't this be generated from remote repo path?

https://github.com/checkstyle/checkstyle/blob/master/.ci/travis/xtr_releasenotes-gen.sh#L48

Reply:
fixed.